### DISCUSSION TITLE
docker images for arm64

### DISCUSSION THREAD

**Discussion by gtxaspec** [exadel-inc/CompreFace#1164] [Q&A]
are docker images available for arm64?

I see arm64 support was added, but the docker-compose pull's are amd64 only.

**Comment by rpetrina** [comment]
Seconded... Need single image for ARM for Home Assistant on an ARM device. Referring to https://github.com/exadel-inc/CompreFace/pull/1018 for ARM support. See also https://github.com/jakowenko/double-take-hassio-addons/issues/8.
